components/expenses/PayoutBankInformationForm.js

```diff
@@ -21,6 +21,8 @@ import { P } from '../Text';
 const formatStringOptions = strings => strings.map(s => ({ label: s, value: s }));
 const formatTransferWiseSelectOptions = values => values.map(({ key, name }) => ({ label: name, value: key }));
 
+const { TW_API_COLLECTIVE_SLUG } = process.env;
```

Comment:
I'm not sure about this syntax, are we sure it's properly processed by webpack?

I learned to avoid it. See: https://github.com/webpack/webpack/issues/5392

Reply:
@znarf : Oh, we already use it at several other places such as, https://github.com/opencollective/opencollective-frontend/blob/8ce8964bcf02a6a9a6a6eb9e87660b33cfce94e8/components/accept-financial-contributions/AcceptContributionsOurselvesOrOrg.js#L34 and https://github.com/opencollective/opencollective-frontend/blob/27e60421534d832e52bf517b90d376ec36091ea4/components/edit-collective/sections/BankTransfer.js#L23. But that issue is interesting. Do you know any other way of getting this environment variable? Let me try to find an an alternative. 🤔

Reply:
`const TW_API_COLLECTIVE_SLUG = process.env.TW_API_COLLECTIVE_SLUG` would work for sure but it would be great to dig into it and confirm best practice so we can provide guidelines for future developers.

Reply:
@znarf : I've opened a PR for this at https://github.com/opencollective/opencollective-frontend/pull/6441 and also suggested and approach we can use to refer these webpack env variables. 🐼